docs/providers/aws/guide/resources.md

```diff
@@ -81,4 +81,5 @@ We're also using the term `normalizedName` or similar terms in this guide. This
 |ApiGateway::Deployment | ApiGatewayDeployment{randomNumber}                      | ApiGatewayDeployment12356789  |
 |ApiGateway::ApiKey     | ApiGatewayApiKey{SequentialID}                          | ApiGatewayApiKey1             |
 |SNS::Topic             | SNSTopic{normalizedTopicName}                           | SNSTopicSometopic             |
+|SNS::Subscription      | {normalizedFunctionName}SnsSubscription{normalizedTopicName}   | HelloSnsSubscriptionSomeTopic             |
```

Comment:
I thought about the same, but what's the use case / how would this look like?

One function with two SNS events subscribing to the same topic would look like this:

```yml
functions:
  hello:
    handler: handler.hello
    events:
      - sns: arn:aws:sns:us-east-1:XXXXX:test
      - sns: arn:aws:sns:us-east-1:XXXXX:test
```